common/network-common/pom.xml

```diff
@@ -45,6 +45,22 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
```

Comment:
What do you mean?  looking below its used in common/network-common/src/main/java/org/apache/spark/network/util/LevelDBProvider.java